lib/json_spec/indifference.rb

```diff
@@ -0,0 +1,25 @@
+module JsonSpec
+  module Indifference
```

Comment:
I'd like to see this called `OrderIndifference` to be clearer about its function.